pkg/kubernetes/pkg/client/leaderelectionconfig/config.go

```diff
@@ -62,5 +62,5 @@ func BindFlags(l *componentconfig.LeaderElectionConfiguration, fs *pflag.FlagSet
 		"of a leadership. This is only applicable if leader election is enabled.")
 	fs.StringVar(&l.ResourceLock, "leader-elect-resource-lock", l.ResourceLock, ""+
 		"The type of resource object that is used for locking during "+
-		"leader election. Supported options are `endpoints` (default) and `configmap`.")
+		"leader election. Supported options are `endpoints` (default) and `configmaps`.")
```

Comment:
It is a mistake. It is `configmapS` in code:
https://github.com/kubernetes-incubator/service-catalog/blob/99c0644c36b4e5b805bf1864babc98cc6f99e231/vendor/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go#L27-L31

Reply:
This change should be made in kube/kube and brought back into here.

Reply:
Yes, it should :)